### PR TITLE
Add `dmesg.log` to Fuzzer

### PR DESCRIPTION
**Pull request by alexey-milovidov**
### Changelog category (leave one):
- Not for changelog (changelog entry is not required)

See https://s3.amazonaws.com/clickhouse-test-reports/0/bb74db22b388a5284db8383dde5f248361530563/fuzzer_astfuzzermsan/report.html

### CODE REVIEW

**Comment by alexey-milovidov** [comment]
I don't see the file.

**Review by novikd — APPROVED** [review]
LGTM, but you also need to add a link to the report here: https://github.com/ClickHouse/ClickHouse/blob/master/docker/test/fuzzer/run-fuzzer.sh#L394

**Comment by qoega** [comment]
>I don't see the file.

To fix it we need similar pull before run: https://github.com/ClickHouse/ClickHouse/pull/45125
I copy-pasted sqlancer check from ast fuzzer and had the same issue.

**Comment by novikd** [comment]
> To fix it we need similar pull before run: https://github.com/ClickHouse/ClickHouse/pull/45125
I copy-pasted sqlancer check from ast fuzzer and had the same issue.

That's completely unrelated. File exists on s3, but there is no link in the report, because report body is fixed for fuzzer in run-fizzer.sh. All we need is to add one line here:
https://github.com/ClickHouse/ClickHouse/blob/master/docker/test/fuzzer/run-fuzzer.sh#L394

**Comment by alexey-milovidov** [comment]
Ok...

**Comment by azat** [comment]
>I don't see the file.

@alexey-milovidov there are problems with the file permissions:

>[docker/test/fuzzer/run-fuzzer.sh](https://github.com/ClickHouse/ClickHouse/pull/45008/files#diff-928246a77d03f80c240a38a9eb40cb4b9d3792e027198145437428537d544bd0)
 100755 → 100644

And if you will check the report by composing the link [manually](https://s3.amazonaws.com/clickhouse-test-reports/45008/dff5e3eafdeff64363174bbf6933b873f5363a71/fuzzer_astfuzzerdebug/main.log), you can see this:

```
2023-01-13 06:39:56	 timeout: failed to run command ‘/run-fuzzer.sh’: Permission denied
```

And also you can see errors in github actions (because of `set -e`):

```
INFO:root:Run failed
...
INFO:root:Upload /home/ubuntu/actions-runner/_work/_temp/ast_fuzzer_debug/workspace/main.log to 45008/dff5e3eafdeff64363174bbf6933b873f5363a71/fuzzer_astfuzzerdebug/main.log. Meta: {'ContentType': 'text/plain; charset=utf-8'}
INFO:root:Exception uploading file server.log.zst text [Errno 2] No such file or directory: '/home/ubuntu/actions-runner/_work/_temp/ast_fuzzer_debug/workspace/server.log.zst'
INFO:root:Exception uploading file fuzzer.log text [Errno 2] No such file or directory: '/home/ubuntu/actions-runner/_work/_temp/ast_fuzzer_debug/workspace/fuzzer.log'
INFO:root:Exception uploading file report.html text [Errno 2] No such file or directory: '/home/ubuntu/actions-runner/_work/_temp/ast_fuzzer_debug/workspace/report.html'
INFO:root:Exception uploading file core.zst text [Errno 2] No such file or directory: '/home/ubuntu/actions-runner/_work/_temp/ast_fuzzer_debug/workspace/core.zst'
INFO:root:Exception uploading file dmesg.log text [Errno 2] No such file or directory: '/home/ubuntu/actions-runner/_work/_temp/ast_fuzzer_debug/workspace/dmesg.log'
```

File: docker/test/fuzzer/run-fuzzer.sh [hunk 1]
```diff
@@ -368,6 +368,7 @@ if [ -f core.zst ]; then
 fi
 
 rg --text -F '<Fatal>' server.log > fatal.log ||:
+dmesg -T > dmesg.log ||:
```
Comment (azat) [inline docker/test/fuzzer/run-fuzzer.sh:371]: Consider adding `dmesg --clear` at the beginning of the script to avoid overlaps with other containers.

**Comment by alexey-milovidov** [comment]
We have reproduced a bug in GitHub in this pull request.
When you edit a file in the web editor, its mode is reset.

**Pull request merged**